script/hassfest/quality_scale.py

```diff
@@ -1051,7 +1051,6 @@ class Rule:
     "uvc",
     "v2c",
     "vallox",
-    "vasttrafik",
```

Comment:
Please update the quality scale record and check in a separate PR without any other changes, after this PR is merged.